realm/realm-library/src/main/cpp/io_realm_internal_UncheckedRow.cpp

```diff
@@ -375,3 +375,19 @@ JNIEXPORT jlong JNICALL Java_io_realm_internal_UncheckedRow_nativeGetFinalizerPt
     return reinterpret_cast<jlong>(&finalize_unchecked_row);
 }
 
+JNIEXPORT jlong JNICALL Java_io_realm_internal_UncheckedRow_nativeGetBacklinkView
+  (JNIEnv *env, jobject, jlong nativeRowPtr, jlong srcTablePtr, jlong srcColumnIndex) {
+
+    TR_ENTER_PTR(nativeRowPtr)
+    if (!ROW_VALID(env, ROW(nativeRowPtr)))
+        return -1;
+    try {
+        Row* row = ROW(nativeRowPtr);
+        Table* srcTable = TBL(srcTablePtr);
+        TableView backlinkView = row->get_table()->get_backlink_view(row->get_index(), srcTable, srcColumnIndex);
+        TableView* tv = new TableView(backlinkView);
```

Comment:
`TableView* tv = new TableView(std::move(backlinkView));`

It allows the state within `backlinkView` to be moved into the newly-allocated object, rather than having to create a copy of it only to then deallocate the original.